particles/Tutorial/Kotlin/3_JsonStore/JsonStore.arcs

```diff
@@ -0,0 +1,26 @@
+// Tutorial 3: JSON Store
+// Loads data stored in a JSON file.
+
+// Defines a new Entity called PersonDetails, with two fields.
+schema PersonDetails
+  Text name
+  Number age
+
+// Creates a data store of type PersonDetails, named PersonToGreetStore. The data will be loaded from the file data.json.
+store PersonToGreetStore of PersonDetails in 'data.json'
+
+particle JsonStoreParticle in '../build/bin/wasm32/mainDebugExecutable/main.wasm'
+  // This particle has an input parameter called inputData. We can use this parameter in the particle's JavaScript file.
```

Comment:
I like that you are adding context. 
Though, this parameter isn't used in a JS file, it's used in Kotlin.